packages/contracts/contracts/optimistic-ethereum/OVM/execution/OVM_ExecutionManager.sol

```diff
@@ -809,6 +871,56 @@ contract OVM_ExecutionManager is iOVM_ExecutionManager, Lib_AddressResolver {
         );
     }
 
+    /**
+     * @notice Overrides BALANCE.
+     * @param _contract Address of the contract to query the OVM_ETH balance of.
+     * @return _BALANCE OVM_ETH balance of the requested contract.
+     */
+    function ovmBALANCE(
+        address _contract
+    )
+        override
+        public
+        returns (
+            uint256 _BALANCE
+        )
+    {
+        // Easiest way to get the balance is query OVM_ETH as normal.
+        bytes memory balanceOfCalldata = abi.encodeWithSelector(
+            IUniswapV2ERC20.balanceOf.selector,
```

Comment:
I think we should explicitly use `OVM_ETH` here and not the Uniswap interface because we do expect the contract to be *the* `OVM_ETH`.